src/components/blocks.ts

```diff
@@ -220,4 +231,36 @@ export default class Blocks {
   public indexOf(block: Block): number {
     return this.blocks.indexOf(block);
   }
+
+  /**
+   * Fires when Editor is loaded.
+   *
+   * All pending tasks should be processed here.
+   */
+  public onEditorReady(): void {
+    this.editorIsReady = true;
+
+    this.pendingBlocks.forEach((block) => block.call(BlockToolAPI.RENDERED));
+  }
```

Comment:
let's document `@this` context type

Reply:
You could also use TypeScript to add type to `this`:
`public onEditorReady(this: Proxy<Blocks>): void {`